src/unix/linux-core.c

```diff
@@ -107,6 +109,26 @@ int uv__platform_loop_init(uv_loop_t* loop) {
 }
 
 
+int uv__platform_loop_init(uv_loop_t* loop) {
+  return uv__epoll_init(loop, 1);
+}
+
+
+int uv__io_fork(uv_loop_t* loop) {
+  int err;
+  uv__close(loop->backend_fd);
+  if (loop->inotify_fd != -1) {
+    uv__io_stop(loop, &loop->inotify_read_watcher, POLLIN);
+    uv__close(loop->inotify_fd);
+    loop->inotify_fd = -1;
+    err = uv__init_inotify(loop);
+    if (err)
+      return err;
+  }
+
+  return uv__epoll_init(loop, 0);
```

Comment:
Ok, this is coming back to me now that I'm looking at it again. The backend fd is a red herring. The true intent here was that fs event watchers (at least the portable set, file watchers) that were started *before* the fork should be able to receive events *after* the fork, just as with sockets, etc. 

I just realized I didn't have a test for that, so I added one in be3f24cb. It works as expected for kqueue platforms (os X at least), but actually doesn't work (times out) on Linux. I thought that preserving `loop->inotify_watchers` would be enough, but that's clearly not the case.